### DISCUSSION TITLE
[blocks]: Please add back the previous block examples as well

### DISCUSSION THREAD

**Discussion by JayMGurav** [shadcn-ui/ui#5609] [Blocks Request]
### Description

There were few blocks showcasing the order listing page, and few other ecom blocks as well, Please add them back.

### Example

<img width="776" alt="image" src="https://github.com/user-attachments/assets/5a12a5a5-2756-4f17-b64d-d6666eb32ec4">

**Comment by krabhi1** [comment]
it's  [here](https://ui.shadcn.com/)

**Reply by felipe-lima-coelho** [reply]
Where? Now they only have the sidebar and login/registration blocks.

**Reply by felipe-lima-coelho** [reply]
> > Where? Now they only have the sidebar and login/registration blocks.
> 
> https://ui.shadcn.com/examples/forms

You know that this is not the core of my question, right? The blocks that previously existed are no longer available. We are not referring to the old "examples" or the new "blocks", but rather to the old "blocks" that have been removed. Do you understand?
